### ISSUE TITLE
TIM using Lat-Lon offset values failed ASN.1 encoding

### ISSUE THREAD

**Issue by richardlsmith** [usdot-jpo-ode/jpo-ode#372]
We submitted a TIM following the format for a path defined by Lat,Lon offset nodes. The ODE logs show a failed ASN.1 encoding error when processing this TIM. Attached is the submitted TIM json file, and the ODE log showing the error when processing this TIM. 

[offset_tim.txt](https://github.com/usdot-jpo-ode/jpo-ode/files/4495262/offset_tim.txt)
[ode_log2.txt](https://github.com/usdot-jpo-ode/jpo-ode/files/4495264/ode_log2.txt)

We generated the submitted TIM json file, based on the documentation https://usdot-jpo-ode.github.io/#api-TIM-timPost, however there were three elements of this test TIM definition that were unclear. 

1)	In the documentation, the “type” element in the region/path appears to correspond with the “offset” element definition from the J2537 standard. Based on the standard, it appears that the “type” element should be set to “ll” to support Lat.Lon offset nodes. The documentation does not specify what value to use for the “type” element, so we used the “ll” value in our test TIM.

2)	The documentation describes the Lat,Lon offset values as being “in degrees from the anchor point”.  For our test TIM we are using an open path with a single, initial anchor point. The J2735 standard describes the sequence of nodes in a path as having a latitude/longitude offset from the “last” point. In our test TIM we created each node in the path as an offset in latitude/longitude from the previous node in the path. Only the first node in our path is offset from the “anchor” point.

3)	In the documentation, the “scale” element in the region/path appears to correspond with the scale element of the J2537 standard. According to the standard this “scale” element defines a zoom level that is applied to scale the size of the offset difference between nodes. It appears from the standard, that the base value of this zoom level should be 1. If the scale element is not included in the submitted TIM json, it appears that the scale value is defaulted to 0. We specified a region/path/scale value of 1 in our test TIM.

**Comment by richardlsmith** [comment]
I meant J2735 for all standards referenced.

**Comment by TonyEnglish** [comment]
Here is additional documentation for this feature (including sample input files with fixed code):
https://github.com/usdot-jpo-ode/jpo-ode/issues/257

here is the export from Jira for the code description to add relative TIM locations:
[ODE-781.doc.pdf](https://github.com/usdot-jpo-ode/jpo-ode/files/4530495/ODE-781.doc.pdf)

**Comment by snallamothu** [comment]
[node_ll_Log.txt](https://github.com/usdot-jpo-ode/jpo-ode/files/4531149/node_ll_Log.txt)
Further notes provided by Brandon - I looked into the JSON that Tony found (issue 257) that had previously worked and cannot get it to work at this point. Version 3 of the ODE now requires the “request” tag to be included so that was the only part of the request that I changed. I’ve attached the resulting logs from the ODE (the TIM JSON can be found in the first line of the log, the ODE prints it upon consumption).
